### ISSUE TITLE
Could not find potions in the belt issue

### ISSUE THREAD

**Issue by LehaKoK** [aeon0/botty#783]
Wrong hotkey in show belt error, but i make it ~ and it didnt work( Close after 10 secs

Where is my mistake plz?

**Comment by derek337** [comment]
ERROR Could not find potions in belt. Most likely due to "show_belt" in params.ini having the incorrect hotkey.

Try changing your hotkey to something else.

**Comment by LehaKoK** [comment]
> ERROR Could not find potions in belt. Most likely due to "show_belt" in params.ini having the incorrect hotkey.
> 
> Try changing your hotkey to something else. And ensure your params are set properly.

Yeah, mate i checked it many times. Time adter time i get that issue and bot stop(

**Comment by dmonti** [comment]
Same problem here, bot found the belt for sereval runs and then sudently this error happens
